docs/lsm/partial_compiles/partial_compiles.rst

```diff
@@ -6,40 +7,63 @@ Partial compilation is an approach to speed up compilation when the Service Inve
 
 Ordinarily, LSM re-compiles all instances on every update. This means that as the inventory grows, the compiles become slower. Partial compiles allow LSM to re-compile only those instances that are relevant to the current service instance, avoiding any slowdown. 
 
+Implementation guidelines
+-------------------------
+
+1. for every :inmanta:entity:`lsm::ServiceEntity`, 
+
+    1. make sure to collect all resources it contains in the relation :inmanta:relation:`owned_resources<lsm::ServiceBase.owned_resources>`
+    2. make sure to always enable the select the ``parent`` implementations (`implement ... using parents`)
```

Comment:
I think a typo / rewrite error snuck in here somewhere: "to always enable the select the". Did you mean this?
```suggestion
    2. make sure to always select the ``parent`` implementations (`implement ... using parents`)
```